src/app/frontend/deploy/deployfromsettings.html

```diff
@@ -140,18 +156,18 @@
       <md-input-container flex>
         <label>CPU requirement (cores)</label>
         <input ng-model="ctrl.cpuRequirement" type="number" name="cpuRequirement" min="0">
-        <ng-messages for="ctrl.form.cpuRequirement.$error" role="alert" multiple>
-          <ng-message when="number">CPU requirement must be a valid number.</ng-message>
-          <ng-message when="min">CPU requirement must be positive.</ng-message>
-        </ng-messages>
+        <div ng-messages="ctrl.form.cpuRequirement.$error" ng-if="ctrl.form.cpuRequirement.$touched">
```

Comment:
I have just checked - the bug I thought it was fixing was actually fixed by ng-if="ctrl.form.name.$touched" (see upcoming answer relevant to this point). This change actually has no effect. I will revert.